test/object_utils_test.rb

```diff
@@ -13,9 +13,9 @@ class ObjectUtilsTest < TestCaseClass
     assert 1.unfriendly_id?
   end
 
-  test "numeric strings are neither friendly nor unfriendly" do
+  test "numeric strings are not friendly but are unfriendly" do
     assert_equal nil, "1".friendly_id?
-    assert_equal nil, "1".unfriendly_id?
+    assert_equal false, "1".unfriendly_id?
```

Comment:
The three-way logic was actually intentional here, and we shouldn't change it. This is because a string with nothing but integers may in fact be a friendly id. For example, assume a "Car" model that uses the model name as a slug - a Peugeot 206 may have "206" as its slug.